File: Java_MVP/app/src/main/java/com/gotoinc/requesin/view/BaseViewContract.java
package com.gotoinc.requesin.view;

/**
 * Created by Illia Derevianko on 26.11.18.
 * GoTo Inc.
 */
public interface BaseViewContract {
    void onError();
    void onError(String message);
    void onTimeout();
    void onNetworkError();
}

File: Kotlin_MVP/app/src/main/java/com/gotoinc/requesin/view/BaseViewContract.kt
package com.gotoinc.requesin.view

/**
 * Created by Illia Derevianko on 28.11.18.
 * GoTo Inc.
 */
interface BaseViewContract {
    fun onError()
    fun onError(message: String)
    fun onTimeout()
    fun onNetworkError()
}